clients/linux/src/app.rs

```diff
@@ -694,69 +696,105 @@ impl LbApp {
     fn show_dialog_import_file(
         &self, parent: Uuid, uris: Vec<String>, finish_ch: Option<glib::Sender<Vec<String>>>,
     ) -> LbResult<()> {
-        let (d, disk_lbl, lb_lbl, prog_lbl, pbar) = self.gui.new_import_export_dialog(true);
+        let d = self.gui.new_dialog("Import Files");
+        util::gui::set_marginy(&d, 36);
+        util::gui::set_marginx(&d, 100);
+
+        let disk_lbl = GtkLabel::new(None);
+        util::gui::set_marginx(&disk_lbl, 16);
+        disk_lbl.set_margin_top(5);
+
+        let pbar = GtkProgressBar::new();
+        util::gui::set_marginx(&pbar, 16);
+        util::gui::set_marginy(&pbar, 16);
+        pbar.set_size_request(300, -1);
+
+        let prog_lbl = GtkLabel::new(None);
+        util::gui::set_marginy(&prog_lbl, 4);
+
+        d.get_content_area().add(&disk_lbl);
+        d.get_content_area().add(&pbar);
+        d.get_content_area().add(&prog_lbl);
 
         const FILE_SCHEME: &str = "file://";
 
-        let mut total = 0;
+        let total = Cell::new(0);
         let progress = Rc::new(RefCell::new(-1));
```

Comment:
maybe we want to be uniform with types and initial values here?

Reply:
How do you mean? As in `progress` should also be a `Cell`?